src/schema/posts.ts

```diff
@@ -1411,8 +1399,8 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         info,
         (builder) => {
           const queryBuilder = builder.queryBuilder.where(
-            `"${builder.alias}"."id" = :id AND ${builder.alias}."sourceId" = :sourceId`,
-            { id, sourceId },
+            `"${builder.alias}"."id" = :id`,
```

Comment:
The id on its own is a unique identifier, no need for any other params. Probably would just make the query worse, although not significant.